isis/src/docsys/build/redirect.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<redirectPage>
+  <div class="isisNavigationBar">
+  <table>
+    <tr>
+      <td><img src="assets/banners/banner.jpg" width="600" height="106" alt="[USGS Banner]" border="0" usemap="#bannermap"/>
+        <map name="bannermap" id="bannermap">
+          <area shape="rect" target="_top" alt="Visit the USGS Home Page" coords="2,1,186,73" href="http://www.usgs.gov"
+            title="Visit the USGS Home Page"/>
+          <area shape="rect" target="_top" alt="Go to the Astrogeology Research Program Home Page" coords="2,75,249,102"
+            href="http://astrogeology.usgs.gov"   title="Go to the Astrogeology Research Program Home Page"/>
+          <area shape="default" alt="USGS" nohref="nohref"/>
+        </map>
+      </td>
+    </tr>
+  </table>
+  </div>
+
+  <meta http-equiv="refresh" content="0; url='7.0.0/index.html'" />
```

Comment:
We should auto-generate the version number here so that it's not another thing we have to keep updating